panels/export_model.py

```diff
@@ -457,6 +457,21 @@ def make_mesh_data(context, export_mesh_groups, ssbh_skel_data):
             list of potential issues that need to validate
             1.) Shape Keys 2.) Negative Scaling 3.) Invalid Materials 4.) Degenerate Geometry
             '''
+
+            # Check if any of the faces are quads, and converts them into tris
+            if any(len(f.vertices) == 4 for f in mesh.data.polygons):
+                # https://blender.stackexchange.com/questions/45698
+                me = mesh.data
```

Comment:
This should work on the mesh copy to avoid modifying the original mesh. You'll need to pull the latest commit from main to resolve the merge conflicts.